pom.xml

```diff
@@ -5,14 +5,16 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.40</version>
+    <version>3.46</version>
     <relativePath />
   </parent>
   <artifactId>display-url-api</artifactId>
-  <version>2.3.2-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
 
   <properties>
+    <revision>2.3.2</revision>
+    <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.89.4</jenkins.version>
```

Comment:
2.121.x would be better, since that an implied dependency less

Reply:
Good point, I've just bumped to `2.121.3`